spec/features/visitors/home_spec.rb

```diff
@@ -51,5 +51,12 @@
       expect(current_path).to eq(root_path)
       expect(page).to have_content('A message with a confirmation link has been sent to your email address. Please follow the link to activate your account.')
     end
+
+    it 'I do not see a profile button in the nav bar' do 
+
+      visit root_path
+
```

Comment:
should the sad path be that the profile button isn't there, or that if you hit it redirects you to the sign in page?

Reply:
I did !! lol